lib/bdev/nvme/blockdev_nvme.c

```diff
@@ -375,24 +380,50 @@ blockdev_nvme_dump_config_json(struct spdk_bdev *bdev, struct spdk_json_write_ct
 	spdk_json_write_name(w, "firmware_revision");
 	spdk_json_write_string(w, buf);
 
+	spdk_json_write_object_end(w);
+
+	spdk_json_write_name(w, "vs");
+	spdk_json_write_object_begin(w);
+
 	spdk_json_write_name(w, "nvme_version");
 	if (vs.bits.ter) {
 		spdk_json_write_string_fmt(w, "%u.%u.%u", vs.bits.mjr, vs.bits.mnr, vs.bits.ter);
 	} else {
 		spdk_json_write_string_fmt(w, "%u.%u", vs.bits.mjr, vs.bits.mnr);
 	}
 
-	spdk_json_write_name(w, "nsid");
+	spdk_json_write_object_end(w);
+
+	spdk_json_write_name(w, "csts");
+	spdk_json_write_object_begin(w);
+
+	spdk_json_write_name(w, "rdy");
+	spdk_json_write_string_fmt(w, "%u", csts.bits.rdy);
```

Comment:
These should probably be JSON bools or numbers rather than strings unless there's a good reason to use a string.

I'm also not totally convinced that reporting RDY is useful, since it should always be set during runtime (by the time a bdev is reported).  Similarly, for SHST, we should never be shutting down the controller while it is still exposed as a bdev.